src/styles/components/quota/styles.less

```diff
@@ -15,6 +15,10 @@
       margin-bottom: @base-spacing-unit;
     }
 
+    .unit {
+      font-weight: @quota-main-font-weight;
```

Comment:
where is `unit` used? maybe it's gone with some iteration?

if we don't need it anymore, we don't seem to need `@quota-main-font-weight` as well.

Reply:
It's fixing a bug that @mperrotti noticed in another PR - https://github.com/dcos/dcos-ui/pull/3968#pullrequestreview-260404171 .